src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,44 @@
+import "./sidebar.css";
+import { NavLink } from "react-router-dom";
+import { AiOutlineHome, FaRegUser, FiSettings } from "../../assets/icons";
+const sidebar = [
+  {
+      id: 1,
+    name: "Home",
+    icon: <AiOutlineHome></AiOutlineHome>,
```

Comment:
suggestion: <AiOutlineHome></AiOutlineHome> can be shortened to <AiOutlineHome/> so that it looks nice to read